SU2_CFD/src/solvers/CIncEulerSolver.cpp

```diff
@@ -3016,11 +3007,8 @@ void CIncEulerSolver::SetResidual_DualTime(CGeometry *geometry, CSolver **solver
       if (implicit) {
         su2double delta = (second_order? 1.5 : 1.0) * Volume_nP1 * Density / TimeStep;
 
-        for (iDim = 0; iDim < nDim; iDim++)
+        for (iDim = 0; iDim <= nDim; iDim++)
           Jacobian.AddVal2Diag(iPoint, iDim+1, delta);
```

Comment:
I think this makes the intent clearer, apply to the rest of the function please.
```suggestion
        for (iVar = 1; iVar < nVar; ++iVar)
          Jacobian.AddVal2Diag(iPoint, iVar, delta);
```